domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/workbasket/WorkBasketUserDefaultService.java

```diff
@@ -65,9 +64,7 @@ private void putUserProfiles(final List<WorkBasketUserDefault> workBasketUserDef
 
         } catch (Exception e) {
             LOG.error("Problem updating user profile: {} because of ", workBasketUserDefaults, e);
-            throw new ServiceException(String.format("Problem updating user profile: '%s' because of '%s'",
```

Comment:
Will we loose this detail from eventual API responses? If so, such loss of detail may slightly harm the API Client's experience.

Reply:
The exception block required only for http call, and downstream status code will be enough, as we are not throwing solid 500 status code anymore.